c2corg_api/views/user.py

```diff
@@ -53,6 +54,8 @@ def validate_json_password(request):
         # later on requires plain string otherwise it raises
         # the "Unicode-objects must be encoded before hashing" error.
         password = request.json['password']
+        if len(password) < 3:
+            raise 'Password too short'
```

Comment:
Why are you doing this check in the `try except` block? It would maybe cleaner to write:
```
    if 'password' not in request.json:
        request.errors.add('body', 'password', 'Required')
        return # note the return!

    password = request.json['password']
    if len(password) < 3:
        request.errors.add('body', 'password', 'Password too short')
        return
    try:
        # We receive a unicode string. The hashing function used
        # later on requires plain string otherwise it raises
        # the "Unicode-objects must be encoded before hashing" error.
        request.validated['password'] = password.encode(ENCODING)
    except:
        request.errors.add('body', 'password', 'Invalid')
```

That way it is clearer what raises an exception.